seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/BaseServlet.java

```diff
@@ -145,4 +155,36 @@ protected Map<String, String> getParameterMap(HttpServletRequest req) {
         }
         return reqParameterMap;
     }
+
+    @VisibleForTesting
+    public void writeJsonForTest(HttpServletResponse resp, Object obj, String type)
+            throws IOException {
+        switch (type) {
+            case "1":
+                writeJson(resp, obj);
+                break;
+            case "2":
+                writeJson(resp, (JsonArray) obj);
+                break;
+            case "3":
+                writeJson(resp, (JsonObject) obj);
+                break;
+            case "4":
+                writeJson(resp, (JsonArray) obj, 200);
+                break;
+            case "5":
+                writeJson(resp, (JsonObject) obj, 200);
+                break;
+            case "6":
+                writeJson(resp, obj, 200);
+                break;
+            case "7":
+                write(resp, obj);
+                break;
+            case "8":
+                writeHtml(resp, obj);
+                break;
+            default:
+        }
+    }
```

Comment:
Please do not add intrusive test code in the source code. We can verify the get method response of different servlets. For example https://github.com/apache/seatunnel/blob/e01108a0ef129e7ebb49ed031d85966773713c9b/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/servlet/CurrentNodeLogServlet.java#L46